data/helpers.d/network

```diff
@@ -47,7 +47,7 @@ ynh_find_port () {
 # | arg: path - the web path to check the availability of
 ynh_webpath_available () {
 	local domain=$1
-	local path=$(ynh_normalize_url_path $2)
+	local path=$2
```

Comment:
In my point of vue, `ynh_normalize_url_path` should be used only before the registering of the path, in the install script.
Because, this helper is designed only to prevent a foolish error. Just to prevent to crash the script for a small error.

But, this path should be fixed immediately, that is why I think we should use this helper only at the beginning of the install script.